cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/ProgramLifecycleHttpHandler.java

```diff
@@ -1402,12 +1413,26 @@ private AppFabricServiceStatus stop(Id.Program identifier, ProgramType type) {
   }
 
   private void getRuns(HttpResponder responder, Id.Program programId, String status,
-                       long start, long end, int limit) {
+                       long start, long end, int limit, final ProgramType programType) {
```

Comment:
Ah you are right, did not know that. Will update to get it from the Program ID instead